### PR TITLE
Leangseu edx/fix-au-69

### PR DESCRIPTION
**Pull request by leangseu-edx**
This is an improvement for [this pr](https://github.com/edx/edx-ora2/pull/1664). It includes fail safe.

- [x] Tested on `sandbox`

**TL;DR -**

Improvement to developing javascript using webpack build:
- [x] include source map
- [x] has distinct name on production build
- [x] `npm run build` to build production
- [x] `npm run dev` to build development
- [x] `npm run start` to start development server
- [x] `make static` should generate javascript for `dev/prod` build

Bonus:
- [x] Hot reload in dev build. **CMS** does not hot reload yet.

JIRA: [educator 5480](https://openedx.atlassian.net/browse/EDUCATOR-5480)

**What changed?**

- `make static` should generate generate javascript for `dev/prod` build
- breakpoint, sourceMap, debugger should work on `npm run build` or `npm run start`
- `npm run start` should start dev server

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

** Hot Reload **
- from local directory (not in lms shell)
- `npm run start`
- from devstack directory (not in lms shell)
- `make lms-restart studio-restart` This is only require when you `load_static.py` to reload its static method.
  - **NOTE**: cms does not support hot reload at the moment
- modify `javascript` or add `debugger` within lms page that has `xblock-ora`. It should hot reload the `xblock`.
  - **NOTE**: lms use iframe while cms is not. One of the reason hot reload not working properly.

**Reviewer Checklist**
- [x] `make static`
- [x] `npm run dev`
- [x] `npm run start`

NOTE:
- Each command require `make lms-restart studio-restart`
- Source map should be working
- `Debugger` should only work on development build

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/xblock/load_static.py [hunk 1]
```diff
@@ -10,6 +10,12 @@
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+def urljoin(*args):
+    begining_slash = '/' if args[0].startswith('/') else ''
```
Comment (jansenk) [inline openassessment/xblock/load_static.py:14]: Definitely add a docstring here explaining why you're doing this

Comment (jansenk) [reply openassessment/xblock/load_static.py:14]: The tests are very clarifying but there should be something here as well

File: openassessment/xblock/load_static.py [hunk 2]
```diff
@@ -10,6 +10,12 @@
...
+    trailing_slash = '/' if args[-1].endswith('/') else ''
+    return begining_slash + '/'.join(map(lambda x: str(x).strip('/'), args)) + trailing_slash
```
Comment (jansenk) [inline openassessment/xblock/load_static.py:16]: maybe you could also pull the lambda out into its own line

**Review by jansenk — COMMENTED** [review]
This looks good to me, I just don't think it's clear at first glance what urljoin is doing

**Review by jansenk — APPROVED** [review]
LGTM

**Comment by jansenk** [comment]
After this merges let's spool up another sandbox just to be safe
